src/test/Testcafe/page-objects/common.js

```diff
@@ -48,7 +48,7 @@ export default class CommonPage {
   }
 
   async returnToHomePage() {
-    repoProfileHomeButton = Selector("#navbar > div.item.brand > a > img"); //the home button, should the user be on the repository profile page
+    var repoProfileHomeButton = Selector("#navbar > div.item.brand > a > img"); //the home button, should the user be on the repository profile page
```

Comment:
use `let` instead of `var`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let

Reply:
Fixed